acquire/tools/decrypter.py

```diff
@@ -480,7 +494,7 @@ def main():
             elif not all(successes):
                 exit_code = 2
             # Else, if all were successful but there were still tasks to handle, return 2
-            elif all(success) and tasks:
+            elif success and tasks:
```

Comment:
You are absolutely right. 

I think the original author meant `all(successes)`. 
This is superfluous, because we know at this point that `not all(successes) == False`,
which implies that `all(successes) == True`.

I also changed the exit_code to 3 in this case. 
Not sure what the author intended, but otherwise he would probably have written `not all(sucesses) or tasks` in the second condition.
(Arbitrary in any case).

Reply:
Indeed, not `all(successes)` is not needed there, I think the `all(success)` is a residu of changing code around. At the end, you just need to check if there are still tasks to be handled, while the handled tasks are all successful.

I also agree with changing the exit code to 3.